### ISSUE TITLE
exportPNG in CindyScript

### ISSUE THREAD

**Issue by BernhardWerner** [CindyJS/CindyJS#919]
Currently, `exportPNG()` always names the downloaded file "CindyJSExport". I would like to add the option to give the file a custom name. Before starting to work on that, any objections to adding this function directly to CindyScript? Or are we content with calling `javascript("cindy.exportPNG()");` ? 

PS: I would, of course, keep the function on the JS side. But I think it would be nice to have both.
PPS: I would do the same for all other export functions: PDF and SVG, as far as I am aware of.

**Comment by kortenkamp** [comment]
I do not think it is a good idea to add this directly to CindyScript for two reasons: If called in an interactive event like `csdraw` or `csmousemove` this would cause chaos for the user, and on the desktop (in Cindy Classic) it is not clear how this should behave (does it write to a file? This would need open a FileBrowser at least on macOS for security reasons…). And finally, please keep in mind that there might be several ports open connected to the same kernel, in that case we do not know which port to export…

So, this is a welcome new feature, but at least I prefer to see it in javascript, and you can call it via `javascript(…)` in case you really want to trigger it from CindyJS.

**Comment by BernhardWerner** [comment]
Ah, that makes sense!
